blueprints/addon/files/README.md

```diff
@@ -10,12 +10,15 @@ Installation
 ember install <%= addonName %>
 ```
 
-
 Usage
 ------------------------------------------------------------------------------
 
 [Longer description of how to use the addon in apps.]
 
+Compatibility
+------------------------------------------------------------------------------
+
+It is recommended to use Ember 2.18 or anything higher than that for this addon to work properly.
```

Comment:
I'm wondering if prose is the right form for that. I was thinking maybe a list of bullet points would be more suited here. Something like:

- Ember.js v2.18 or above
- Ember CLI v3.5 or above
- `ember-intl` v4.x